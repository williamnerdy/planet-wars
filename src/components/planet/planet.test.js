import React from 'react';
import { shallow } from 'enzyme';
import Planet from './planet';

describe('<Planet />', () => {
  it('should render correctly', () => {
    const element = shallow(
      <Planet
        name="testeeee"
        population="testeeee"
        climate="testeeee"
        terrain="testeeee"
        feature={1}
      />
    );
    expect(element).toMatchSnapshot();
  });
});
