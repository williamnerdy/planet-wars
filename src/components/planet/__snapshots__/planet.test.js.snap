// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Planet /> should render correctly 1`] = `ShallowWrapper {}`;
